.eslintrc.js

```diff
@@ -65,6 +60,9 @@ module.exports = {
     'no-var': ERROR,
     strict: ERROR,
 
+    // Enforced by Prettier
+    'max-len': OFF,
```

Comment:
I was getting some weird syntax error in a transitive module. Google didn't turn up anything so I decided to turn it off since the length is enforced by Prettier, anyway.

Reply:
Prettier won’t warn for long strings tho, and won’t rewrite them either. Which... may be fine?

Reply:
fwiw, this also yelled at me for long line comments, which prettier doesn't touch

Reply:
I'll add a comment so the next person who looks at it knows why I turned it off.